Add register output format

kehh opened the pull request.

kehh committed:
Add register output format

leftclickben commented on app/printTemplates/results/local/register.php:
```diff
@@ -0,0 +1,104 @@
+					print (strlen($vs_display_value) > 100 ? strip_tags(substr($vs_display_value, 0, 97)) . "&hellip;" : $vs_display_value);
+					?></td><?php
+					$vn_line_count ++;
+					error_log($vs_display_value);
```
Your logs are showing ;-)

leftclickben commented on app/printTemplates/results/local/register.php:
```diff
@@ -0,0 +1,104 @@
+
+	while ($vo_result->nextHit()) {
+		if($vn_line_count === 0){
+			print "<thead><tr>";
```
I tend to prefer using either `?>html<?php` or `print "html"` (or `echo "html"`) consistently within the same file, but happy to merge this as-is if you don't want to change it.

leftclickben commented on app/printTemplates/results/local/register.php:
```diff
@@ -0,0 +1,104 @@
+
+				foreach ($va_display_list as $vn_placement_id => $va_display_item) {
+					?><td><?php
+					$bundle_class = preg_replace('/W+/', '_',$va_display_item['bundle_name']);
```
`/W+/g` or you'll only get the first one

leftclickben replied:
Oh wait sorry this is php, my bad.  Ignore previous.

leftclickben commented on app/printTemplates/results/local/register.php:
```diff
@@ -0,0 +1,104 @@
+		if($vn_line_count === 0){
+			print "<thead><tr>";
+			foreach ($va_display_list as $vn_placement_id => $va_display_item) {
+				print "<th class='displayHeader'>" . $va_display_item['display'] . "</th>";
```
This is also generating inconsistent output wrt attribute value quoting style `'` here vs `"` below.  But again, just a style thing, it works either way, so I'm happy to ignore it.

leftclickben commented:
The changes themselves look fine, apart from the `error_log`s (and style issues aside).  However I'm wondering if this is the place to put this.  We've got `providence-configuration` repo, we can put it in there, and symlink it, or better, if there is a constant for it, define the value in `setup.php` like we have with the config files (`__CA_LOCAL_CONFIG_DIRECTORY__`).

kehh committed:
Merge branch 'master-fix' of https://github.com/rwahs/providence into registerReport

kehh committed:
Tidy up register template, remove unused variable and console logs

kehh commented:
As discussed, there are a number of `local` directories (one for each directory in `app/printTemplates`). For simplicity sake we'll keep them in the RWAHS fork.

I have removed the logs and addressed your comments about tag styles.
